runtime-modules/working-group/src/lib.rs

```diff
@@ -309,6 +311,11 @@ decl_storage! {
 
         /// Worker exit rationale text length limits.
         pub WorkerExitRationaleText get(worker_exit_rationale_text) : InputValidationLengthConstraint;
+
+        /// Map member id by hiring application id.
+        /// Required by StakingEventsHandler callback call to refund the balance on unstaking.
+        pub MemberIdByHiringApplicationId get(fn member_id_by_hiring_application_id):
```

Comment:
Why is this needed? It was not needed in the content working group?

The only place I see it used is in `refund_working_group_stake`, in order to do this

```
let member_id = Module::<T, I>::member_id_by_hiring_application_id(hiring_application_id);
```

But you could already resolve the member id from an application by just looking at `Application::member_id`?

Reply:
The similar functionality in the content working group is here:

`pub UnstakerByStakeId get(unstaker_by_stake_id) config(): linked_map StakeId<T> => WorkingGroupUnstaker<LeadId<T>, CuratorId<T>>;`

But we have no `stake_id` in the working group. It is hidden inside the hiring module.

The method like this can be found in the runtime crate implemented within `ContentWorkingGroupStakingEventHandler`.

Reply:
I think I understand.